yarn.lock

```diff
@@ -8776,6 +8778,11 @@ just-extend@^4.0.2:
   resolved "https://registry.yarnpkg.com/just-extend/-/just-extend-4.2.1.tgz#ef5e589afb61e5d66b24eca749409a8939a8c744"
   integrity sha512-g3UB796vUFIY90VIv/WX3L2c8CS2MdWUww3CNrYmqza1Fg0DURc2K/O4YrnklBdQarSJ/y8JnJYDGc+1iumQjg==
 
+karma-safari-launcher@^1.0.0:
+  version "1.0.0"
+  resolved "https://registry.yarnpkg.com/karma-safari-launcher/-/karma-safari-launcher-1.0.0.tgz#96982a2cc47d066aae71c553babb28319115a2ce"
+  integrity sha512-qmypLWd6F2qrDJfAETvXDfxHvKDk+nyIjpH9xIeI3/hENr0U3nuqkxaftq73PfXZ4aOuOChA6SnLW4m4AxfRjQ==
+
```

Comment:
This strikes me as an odd addition, although it's derived from dexie. Any clue as to why dexie needs it?

Reply:
Good catch, I didn't pay any attention to that. Dexie indeed doesn't need it, and in fact it was dropped in a patch version update that was published after I did the update. I'll get rid of it.

Reply:
[Updated Dexie](https://github.com/ebkr/r2modmanPlus/pull/1261/commits/620a87bf05f37dff46ce97292015c12e2b91c5d8), no more extra dependencies.